### DISCUSSION TITLE
Can't start train model

### DISCUSSION THREAD

**Discussion by PxCPeter2801** [PaddlePaddle/PaddleOCR#12681] [Q&A]
I try to train model detection handwritten English but I met this issue:
command: python "E:\PPOCR\PaddleOCR\tools\train.py" -c "E:\PPOCR\PaddleOCR\configs\det\det_mv3_db.yml"
W0219 09:15:38.875943  1948 gpu_resources.cc:96] The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 90 , it is recommended to install the corresponding wheel package according to the installation information on the official Paddle website.
W0219 09:15:38.876183  1948 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 12.4, Runtime API Version: 12.0
W0219 09:15:38.879211  1948 gpu_resources.cc:164] device: 0, cuDNN Version: 8.9.
[2024/02/19 09:15:39] ppocr INFO: train dataloader has 32 iters
[2024/02/19 09:15:39] ppocr INFO: valid dataloader has 172 iters
[2024/02/19 09:15:39] ppocr INFO: load pretrain successful from E:\PPOCR\PaddleOCR\pretrain_models\MobileNetV3_large_x0_5_pretrained
[2024/02/19 09:15:39] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 100 iterations
W0219 09:15:55.513259 14228 system_allocator.cc:231] cudaHostAlloc failed.
W0219 09:15:55.527786 14228 naive_best_fit_allocator.cc:387] cudaHostAlloc Cannot allocate 26214400 bytes in CUDAPinnedPlace

**Comment by tink2123** [comment]
> cudaHostAlloc failed.

According to the information in the log, it may be due to insufficient memory. It is recommended to use system monitoring tools (such as top, htop or Windows Task Manager) to check the current memory usage. If memory usage is high, try closing some applications or processes to free up memory.
